vertx/feign-vertx/src/main/java/feign/VertxFeign.java

```diff
@@ -307,22 +277,22 @@ public Builder timeout(long timeout) {
     }
 
     /**
-     * Defines operation to execute on each {@link HttpClientRequest} before it is sent. Used to
-     * make setup on request level.
+     * Defines operation to execute on each {@link HttpRequest} before it sent. Used to make setup
+     * on request level.
      *
      * <p>Example:
      *
      * <pre>
      * var client = VertxFeign
      *     .builder()
      *     .vertx(vertx)
-     *     .requestPreProcessor(req -&#62; req.putHeader("version", "v1"));
+     *     .requestPreProcessor(req -&#62; req.ssl(true));
      * </pre>
      *
      * @param requestPreProcessor operation to execute on each request
      * @return updated request
      */
-    public Builder requestPreProcessor(UnaryOperator<HttpClientRequest> requestPreProcessor) {
```

Comment:
the ask here would be the same, not sure if viable at all

Reply:
Idem. Current version from core project was never released.

Reply:
Ow, sweet, perfect time to change then =)